packages/pds-ember/addon/components/pds/page/stories/index.stories.js

```diff
@@ -0,0 +1,85 @@
+import hbs from 'htmlbars-inline-precompile'
+import DocsPage, { TITLE } from '../docs.mdx'
+
+export default {
+  title: TITLE,
+  component: 'PdsPage',
+  subcomponents: {
```

Comment:
Should this get added to PageHeader too? Would be nice to see all of the sub components even if there aren't any args.

Reply:
I added it here because `<P.Header>` has arguments that are useful in the Page context.  PageHeader subcomponents are just wrappers for standard markup (no arguments).  Optimally, Storybook would generate docs from `@yield` tags in JS comments, but it doesn't do anything with the data.  However, I've been making sure to show subcomponent use in the code snippet under "Usage" in the docs.  Additionally, `subcomponents` doesn't make sense for template-only subcomponents, because there's no JS to define the `@class`.